presentation/support/presentation.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+
+<meta charset="utf-8">
+<link rel="author" title="Intel" href="http://www.intel.com">
+<link rel="author" title="He Yue" href="mailto:yue.he@intel.com">
+<link rel="help" href="http://w3c.github.io/presentation-api/#interface-presentationconnectionlist">
+<script>
+  var addConnection = function(connection) {
+    connection.onconnected = function () {
+      this.onmessage = function (evt) {
+        this.send(evt.data);
+      };
+  };
+  navigator.receiver.connectionList
+    .then(function(list) {
+      list.onconnectionavailable = function(connections) {
+        addConnection(connections[connections.length-1]);
```

Comment:
Put spaces before and after `-`. Others look good to me.

By the way, what's your testing result?